### PR TITLE
V20 CNV update part2: focal cnv and oncoprint rerun

### PR DESCRIPTION
**Pull request by kgaonkar6**
#### What scientific question is your analysis addressing?
Just a rerun for focal-cn and oncoprint after #1123 

#### What was your approach?
I added the conditions to use base histology in `focal-cn-preparation` module to run for subtyping when needed.

#### What GitHub issue does your pull request address?
#1125

#### Which areas should receive a particularly close look?
NA

#### Is there anything that you want to discuss further?
NA

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### What types of results are included (e.g., table, figure)?
tables

#### What is your summary of the results?
Rerun of focal calls and oncoprint from updated overlap #1123 

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Review by jharenza — APPROVED** [review]
The majority of the goi oncoprints did not change. ATRX deletion was recovered in two HGAT samples, one of which _is_ annotated on pedcbio and we had been missing previously: https://pedcbioportal.kidsfirstdrc.org/patient?studyId=pbta_all&caseId=PT_7WT6P5M8

This looks good to me.

**Comment by kgaonkar6** [comment]
Also wanted to add here that all updates in status in this update were NA in previous CNV calls. 
[change_log.txt](https://github.com/AlexsLemonade/OpenPBTA-analysis/files/6933864/change_log.txt)

```
> change_log %>% select(status_latest,status_previous) %>% replace_na(list(status_latest="NA",status_previous="NA")) %>% table()
               status_previous
status_latest   amplification gain loss  NA
  amplification            10    0    0   3
  gain                      0  370    0  58
  loss                      0    0  126  43
```

File: analyses/focal-cn-file-preparation/02-add-ploidy-consensus.Rmd [hunk 1]
```diff
@@ -45,7 +50,11 @@ if(!dir.exists(output_dir)) {
 # gets included we will have to change the file path here
 consensus_seg_file <- file.path("..", "copy_number_consensus_call", "results", 
                                 "pbta-cnv-consensus.seg.gz")
+if ( params$base_run ==0 ){
 histologies_file <- file.path("..", "..", "data", "pbta-histologies.tsv")
+} else {
+histologies_file <- file.path("..", "..", "data", "pbta-histologies-base.tsv")
+}
```
Comment (jaclyn-taroni) [inline analyses/focal-cn-file-preparation/02-add-ploidy-consensus.Rmd:57]: Per https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1130/files#r686770033:

```suggestion
if ( params$base_run ==0 ){
  histologies_file <- file.path("..", "..", "data", "pbta-histologies.tsv")
  consensus_seg_file <- file.path("..", "..", "data", 
                                  "pbta-cnv-consensus.seg.gz")
} else {
  histologies_file <- file.path("..", "..", "data", "pbta-histologies-base.tsv")
  consensus_seg_file <- file.path("..", "copy_number_consensus_call", "results", 
                                  "pbta-cnv-consensus.seg.gz")
}
```

 I can not include removing the TODO comment in my suggestion for this diff, so let's make sure we do that too!

File: analyses/focal-cn-file-preparation/02-add-ploidy-consensus.Rmd [hunk 2]
```diff
@@ -42,10 +47,15 @@ if(!dir.exists(output_dir)) {
 
 ```{r}
 # TODO: the consensus SEG file is not currently in the data download -- when it
```
Comment (jaclyn-taroni) [inline analyses/focal-cn-file-preparation/02-add-ploidy-consensus.Rmd:49]: Now that the diff has changed, I can suggest removing the TODO 

```suggestion
```

**Comment by jaclyn-taroni** [comment]
If this fixes #1129, I will close #1130

**Comment by jaclyn-taroni** [comment]
Merging based on @jharenza 's review!

**Pull request merged**